src/backend/marsha/bbb/api.py

```diff
@@ -451,7 +452,7 @@ class ClassroomDocumentViewSet(
 
     def get_permissions(self):
         """Instantiate and return the list of permissions that this view requires."""
-        if self.action in ["create", "list", "update", "partial_update"]:
+        if self.action in ["create", "list", "update", "partial_update", "metadata"]:
```

Comment:
`metadata` must be available for more user. For video resource we use this permission `permissions.UserOrResourceIsAuthenticated`

Reply:
I don't get why metadata should be available to students.

Reply:
Because the http OPTION is special and it gives information to the browser https://developer.mozilla.org/fr/docs/Web/HTTP/Methods/OPTIONS

IMO it is dangerous to restrict it like this.